schedule/yast/maintenance/autoyast_create_hdd_maintenance.yaml

```diff
@@ -11,6 +11,7 @@ schedule:
   - autoyast/repos
   - autoyast/logs
   - autoyast/autoyast_reboot
+  - console/scc_deregistration
```

Comment:
deregistration should be where you put it in your VR for example, but not so early before the patching for sure.